### ISSUE TITLE
Author and URL for solutions

### ISSUE THREAD

**Issue by andybalaam** [andybalaam/rabbit-escape#143]
From #120 Solution syntax, provide `author_name` and `author_url` like this:

    solution_author.1=Andy Balaam
    solution_url.1=blah

**Comment by colonelfazackerley** [comment]
It would make sense to switch hints over to be like 
```
hint.1=think
hint.2=more
...
```
at the same time.

**Comment by andybalaam** [comment]
Yes. When I modified the solution code I provided a general mechanism for numbered lists so should be fairly easy.
